README.md

```diff
@@ -3,3 +3,4 @@ Learning github
 Lavanya version 1.1
 Lavanya version 1.2
 Lavanya version 1.3
+Radhakrishna version 1.4
```

Comment:
The version no is wrong, it should be 1.6

Reply:
Hey, I think it should be 1.5